documentation/docs/release-notes/3.0.0.md

```diff
@@ -62,11 +62,16 @@ We've made a significant enhancement to the way API authentication is handled. P
 This transition to service accounts brings fine-grained access control and logging of all actions performed, providing more security, better visibility and auditing capabilities. 
 
 #### Migration from API keys
+
 With this change, API keys are now deprecated. When upgrading from PMM 2 to PMM 3, your existing API keys will be automatically converted to service accounts when you first log in as an Admin user or via CLI.
 For more information see [Migrate to PMM 2 to PMM 3](../pmm-upgrade/migrating_from_pmm_2.md#step-3-migrate-pmm-2-clients-to-pmm-3) and [Service account authentication](../api/authentication.md).
 
 ![Service Accounts page](../images/Service_Accounts.png)
 
+#### CVEs eliminated through architectural changes
+
+The removal of legacy components like Integrated Alerting and DBaaS, combined with the security enhancements above, eliminates multiple potential vulnerabilities and CVEs. This makes PMM v3 our most secure release yet.
```

Comment:
it's not Integrated Alerting, but Alertmanager.

Reply:
as discussed it's about https://perconadev.atlassian.net/browse/PMM-12287